### PR TITLE
Vaccine updates

### PR DESCRIPTION
**Pull request by ar0ch**
TBD

### CODE REVIEW

File: COVID19-Event-Risk-Planner/makeDailyMaps.R [hunk 1]
```diff
@@ -23,27 +25,132 @@ print(current_time)
...
+          county = COUNTY,
+          cnt_fully_vacc = Complete,
+          # cnt_partially_vacc = Partial, # removing partials
+          date = DATE
+      )%>%
+      mutate(
+          
+          date = as_date(date),
+          county = case_when(
+              as.numeric(county) %in% bb_lp ~ 2997,
```
Comment (ar0ch) [inline COVID19-Event-Risk-Planner/makeDailyMaps.R]: @freyja-bt `bb_lp` and `y_ha` refer to objects that don't exist

**Comment by ar0ch** [comment]
@freyja-bt 

Since we're sending all the required geoms at on map load for the No Immunity map, we can't switch out between the two sets of geoms (with and without Joplin and friends).  Can you install just apply the appropriate county vaccine data to these custom city geoms?  If they straddle more than one county, would taking an average be acceptable @sjbeckett?

Otherwise it's looking good (50% vaccinated):

image

File: COVID19-Event-Risk-Planner/makeDailyMaps.R (hunk 1 above)
Comment (freyja-bt) [reply COVID19-Event-Risk-Planner/makeDailyMaps.R]: I'll replace them

Comment (freyja-bt) [reply COVID19-Event-Risk-Planner/makeDailyMaps.R]: I made a new PR with the change

**Comment by freyja-bt** [comment]
Both cities overlap multiple counties, so I'll fix it based on @sjbeckett 's review. I can also split the cities into multiple pieces along the county lines, but that seems like it will be more confusing?

**Comment by sjbeckett** [comment]
My preference would be to keep the risk calculations at the finest scale (including cities).
For vaccination levels, I do expect there to be differences between the city areas and non-city areas of these counties. However, rather than interpolate I think we should move to a coarser scale in these areas -- by computing the vaccination levels of the joint area of Joplin City, Jasper County and Newton County;  and similarly for the joint area of Kansas City, Jackson County, Clay County, Cass County, and Platte County.  We should also add a note describing this to the DataSource.md file. @freyja-bt does this seem a reasonable route forward?

**Comment by freyja-bt** [comment]
Changes recommended by @sjbeckett are in PR #57

**Pull request merged**